### DISCUSSION TITLE
Older Tine20 Packages?

### DISCUSSION THREAD

**Discussion by Matze0811** [tine20/tine20#7262] [Q&A] [answered]
Do anybody still got a 2020.8.2 Package-Version of tine20? 
I've trieb to Upgrade my Installation from 2020.8.2 to ...8.5 and deleted accidentally my earlier downloaded Tine setup files. So i got a Backup of my database (Döner with Backup Funktion of tine20) but ganz go back to a working Installation! Please Help me!

**Comment by lab-at-nohl** [comment, marked as answer]
I may help you.
